inventory/targets/examples/artifactory.yml

```diff
@@ -0,0 +1,45 @@
+# Sources:
+# https://github.com/kapicorp/kapitan/blob/master/docs/compile.md#helm
+# https://artifacthub.io/packages/helm/jfrog/artifactory
+
+classes:
+  - common
+
+parameters:
+  target_name: artifactory
+
+  kapitan:
+    dependencies:
+    - type: https
+      source: https://charts.jfrog.io/artifactory/helm/artifactory-11.7.6.tgz
+      unpack: True
+      output_path: components/charts
+    compile:
+    - input_type: helm
+      input_paths:
+        - components/charts/artifactory
+      output_path: .
+      helm_values:
```

Comment:
I would move this to :

```
parameters:
  artifactory:
    helm:
      values:
        name: artifactory
        <yadayada>
```

so it becomes easier to override things from other places

Reply:
Sure let me update this but I used the reference above which says `helm_values:`